SIANXSVC-806: Anexia vanity import path

beachmachine opened the pull request:
**Changes:**
* Updated documentation to use the vanity import path
* Added a GitHub workflow to update the vanity service on `push`

beachmachine committed:
SIANXSVC-806: Anexia vanity import path

LittleFox94 commented:
maybe change line 1 of `e5e.go` to this:

```go
package e5e // import "go.anx.io/go-e5e"
```

Makes go refuse compiling if imported via another url - see [Golang 1.4 changelog for more](https://go.dev/doc/go1.4#canonicalimports)

LittleFox94 commented:
another idea: `go.anx.io/go-e5e` contains `go` twice - maybe we want to name it `go.anx.io/e5e` instead? Noticed this for go-anxcloud, too, but didn't want to do yet another rename - here we still have that chance :)

beachmachine commented:
> maybe change line 1 of `e5e.go` to this:
> 
> ```go
> package e5e // import "go.anx.io/go-e5e"
> ```
> 
> Makes go refuse compiling if imported via another url - see [Golang 1.4 changelog for more](https://go.dev/doc/go1.4#canonicalimports)

Thanks for the hint! I've added the comment.


> another idea: `go.anx.io/go-e5e` contains `go` twice - maybe we want to name it `go.anx.io/e5e` instead? Noticed this for go-anxcloud, too, but didn't want to do yet another rename - here we still have that chance :)

I've created a pull request to set the target name and changed the import paths here. 
https://github.com/anexia-it/go.anx.io/pull/14
